src/markdown/setup/tableau-server-driver-setup.md

```diff
@@ -0,0 +1,23 @@
+### Tableau Server
+[Link to product webpage](https://www.tableau.com/products/server). 
+
+#### Adding the Amazon DocumentDB JDBC Driver
+[Download](https://github.com/aws/amazon-documentdb-jdbc-driver/releases/latest) the DocumentDB JDBC driver JAR file and copy it to one of these
+directories according to your operating system: 
+   - **_Windows_**: `C:\Program Files\Tableau\Drivers` 
+   - **_Linux_**: `/opt/tableau/tableau_driver/jdbc` 
+
+These directories may need to be created if they do not yet exist. This must be done on each server node.
+
+#### Adding the Tableau connector
+1.  [Download](https://github.com/aws/amazon-documentdb-jdbc-driver/releases/latest) the DocumentDB Tableau connector (a TACO file).
+2.	For each server node:
+      - Create a directory for Tableau connectors if none exists. This needs to be the same path on each machine, and on the same drive that the server is installed on. For example: `C:\tableau_connectors`
```

Comment:
For Windows, you may want to suggest:

For example: (**Windows**) `C:\Users\Public\Documents\tableau_connectors` or (**Linux**) `/srv/tableau_connectors`

At least on Windows, this is guaranteed not to require administration authority to create the directory, were as, a folder in the root directory does need admin privileges.